gen/schema-header.act

```diff
@@ -497,7 +497,7 @@ class DNodeInner(DNode):
         for child in self.children:
             if not isinstance(child, DLeafList):
                 res.append(f"        _{usname(child)} = self.{usname(child)}")
-            child_ns = ", ns='" + child.namespace + "'" if child.namespace != self.namespace else ""
+            child_ns = ", ns='" + child.namespace + "'" if top == True or child.namespace != self.namespace else ""
```

Comment:
with top being a bool, surely we should rely on this sort of implied boolean check, no?

```suggestion
            child_ns = ", ns='" + child.namespace + "'" if top or child.namespace != self.namespace else ""
```

I think we have some deficiencies in boolean evaluation of non-boolean things, but that's not the case here.